package.json

```diff
@@ -10,6 +10,8 @@
   "homepage": "./",
   "license": "MIT",
   "scripts": {
+    "build-seed": "cd ./src/server-browser-scripts && tsc",
+    "run-seed": "node -e 'import(\"./dist/seed/src/server-browser-scripts/seed-data.js\").then(x=> x.resetDeals())' -r dotenv/config . dotenv_config_path=./.env",
```

Comment:
`dotenv/config . dotenv_config_path=./.env`

not possible to use `cross-env DOTENV_CONFIG_PATH` like the other scripts here?

Reply:
I believe I tried it and it doesnt work without all the above.